packages/core/src/Breadcrumb.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import Flex from './Flex'
+import Icon from './Icon'
+import Text from './Text'
+
+const Container = styled(Flex)`
+  cursor: ${props => (props.onClick ? 'pointer' : 'default')};
+`
+
+function Breadcrumb({ className, isCurrent, isCompleted, children, onClick }) {
+  const color = isCurrent || isCompleted ? 'primary' : 'text.light'
+
+  return (
+    <Container alignItems="center" onClick={onClick}>
+      {isCompleted && <CheckIcon />}
+      <Text color={color} bold={isCurrent} fontSize={1}>
```

Comment:
Do we see any utility in making the `fontSize` configurable?

Reply:
Yeah that's a good idea, is there a good way to scale the size of the icon too? I feel like `width/height: themeGet('fontSizes.x')` is a little weird.

Reply:
I think simply adding another prop fontSize with default prop value 1 would serve the configurable suggestion.

Reply:
Yes but it could result in something like:
![image](https://user-images.githubusercontent.com/9325039/64361789-3e35a000-cfdb-11e9-85dd-0d5eecc9cdae.png)
where the icon size is tiny.

I could convert the fontSize to size for the icon too but is it worth it when we should probably be keeping our breadcrumb size consistent across the site???